pytorch_toolkit/formula_recognition/tools/get_config.py

```diff
@@ -0,0 +1,40 @@
+"""
```

Comment:
On the whole file `tools/get_config.py`

I'd propose to move this file and the file `tools/evaluation_tools.py` from the subfolder `tools`/ into a new subfolder `utils/`

You can say that you already has `im2latex/data/utils.py`, and it is a bad decision to have two `utils` in one project -- I will agree, but I would propose to rename    
`im2latex/data/utils.py -> im2latex/data/data.py`

Reply:
File moved